tests/utils.py

```diff
@@ -653,3 +654,12 @@ async def get_data(postgresql_client):
     )
     assert post_ra_one_set == post_ra_two_set
     assert post_ra_one_set
+
+
+def get_as_naive_datetime(timestamp: datetime) -> datetime:
```

Comment:
Do you use this anywhere? I don't see it.

Reply:
It's used in `tests/server/test_environment_metrics.py`

Reply:
For some reason I'd overlooked that it was in `tests/utils.py`, not `src/inmanta/util.py` so I was looking in all the wrong places.